### PR TITLE
Update mob_avail.txt

### PR DESCRIPTION
**Pull request by Balferian**
* **Server Mode**: Pre/Re
* **Description of Pull Request**: 
Added view equipment **"robe"** to **mob_avail.txt**

### CODE REVIEW

File: src/map/mob.cpp [hunk 1]
```diff
@@ -4331,7 +4332,7 @@ static bool mob_readdb_mobavail(char* str[], int columns, int current)
 #endif
 	}
 	else if(columns==3)
-		mob_db_data[mob_id]->vd.head_bottom=atoi(str[2]); // mob equipment [Valaris]
+		mob_db_data[mob_id]->vd.robe = atoi(str[2]); // mob equipment [Valaris]
```
Comment (Lemongrass3110) [inline src/map/mob.cpp]: Why not pet equipment anymore, without updating documentation?

File: src/map/mob.cpp [hunk 2]
```diff
@@ -4311,17 +4311,18 @@ static bool mob_readdb_mobavail(char* str[], int columns, int current)
 	mob_db_data[mob_id]->vd.class_ = sprite_id;
 
 	//Player sprites
-	if(pcdb_checkid(sprite_id) && columns==12) {
+	if(pcdb_checkid(sprite_id) && columns==13) {
```
Comment (Lemongrass3110) [inline src/map/mob.cpp]: Might be better to change this to > 12 and add column 13 as optional inside with a default value.

**Comment by Atemo** [comment]
What about a convertion in yaml format?

**Comment by hannicaldummy** [comment]
I've made all these changes and my map-server keeps saying that the maximum colums at mob_avail is 12, anything? :/

**Comment by aleos89** [comment]
I'm going to go ahead and close this. We have future plans of converting this to YAML which should allow for easier expansions.

**Pull request closed without merging**